framework/include/base/MooseApp.h

```diff
@@ -519,7 +519,13 @@ class MooseApp : public ConsoleStreamInterface,
    *  Whether or not this simulation should only run half its transient (useful for testing
    * recovery)
    */
-  bool halfTransient() const { return _half_transient; }
+  bool testCheckpointHalfTransient() const { return _test_checkpoint_half_transient; }
+  bool halfTransient() const
```

Comment:
What apps use this method? Just griffin?

Reply:
Yup! I'll make another PR to remove the deprecated option once I update Griffin.